Gitlab ci include (#92)
telekom-mms/docker-management-container: README.MD

Examples for build within CI Pipelines can be found under [examples/pipeline](examples/pipeline).

#### Included build preset

> Provided build presets can be found under [pipeline](pipeline). The preset also sets some labels e.g. the version of the dmc release (dmc-version), you can get this information over `docker inspect`.

Reviewer: ```suggestion
Provided build presets can be found under [pipeline](pipeline). The preset also sets some labels e.g. the version of the dmc release (dmc-version), you can get this information with `docker inspect`.
```

``` bash
> docker inspect service-mgmt-dmc --format '{{ json .Config.Labels }}' | jq .
{
  "dmc-version": "3.1.0",
  "org.wakemeops.base_image": "\"docker.io/ubuntu:latest\"",
  "org.wakemeops.commit": "\"65d81642eb025d20c4db5b45758879b379bc6aa1\"",
  "org.wakemeops.maintainers": "\"WakeMeOps <wakemeops.com>\""
}
```

##### [gitlab-ci](pipeline/gitlab-ci/dmc.yml)

##### Inputs

| variable | description | default |
| --- | --- | --- |
| GITHUB_DMC_TAG | version of the dmc release | latest version |
| DMC_NAME | name of the container | dmc |
| DMC_IMAGE | full image name  | `$CI_REGISTRY_IMAGE/${DMC_NAME}` |

> The preset forces an .docker_login as extend. You can define here your docker registry informations.

Reviewer: ```suggestion
The preset needs a `.docker-login`-extend. In it you have to define your docker registry information.
```
